MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -379,9 +379,9 @@ As like `UPC-A`, the `UPC-E` symbology supports only numeric character. It is a
 
 ## EAN-13
 
-The `EAN-13`  standard is based on the `UPC-A` standard.. As like `UPC-A`, it supports only the numeric character. It encodes the 12 digits of input data with the check digit at its end.
+The [`EAN-13`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.EAN13.html) standard is based on the [`UPC-A`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.UPCA.html) standard.. As like [`UPC-A`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.UPCA.html), it supports only the numeric character. It encodes the 12 digits of input data with the check digit at its end.
```

Comment:
You have removed space

Reply:
The space is reflected in the correction part already. will correct this unwanted changes in future sir